### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by ksigei**
**In this milestone;**
- I made sure that there are no linter errors
- I made sure that you used correct workflow
- I made sure that my work was documented
- I used media queries in my CSS file to make my portfolio's desktop version 
- I used  Grid and Flexbox lay out the elements on the page .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,261 +9,393 @@
...
-                        <li><span>Ruby on Rails</span></li>
-                        <li><span>JavaScript</span></li>
-                    </ul>
+             
+         </section>
+         <footer>
+            <div class="footer-container">
+                <div class="header-img-left">
+                    <img src="images/image-geometry_6.png" alt="image">
                 </div>
```
Comment (keroloussamy) [inline index.html:363]: - [ ] Kindly this image is related to the desktop version and it appears in the mob version.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -9,261 +9,393 @@
...
-                    <li>Bootstrap</li>
-                    <li>Ruby on rails</li>
-                    <li>RSpec</li>
-                    <li>Capibara</li>
-                    <li>Selenium</li>
-                </ul>
-    
+                        <textarea name="message" class="msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <button class="btn-collabo-desk" type="submit" name="submit">Start collaboration</button>
+
```
Comment (keroloussamy) [inline index.html:375]: - [ ] Kindly fix the button design to be like the Figma design in the mob version.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @ksigei, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/main.css [hunk 3]
```diff
@@ -462,3 +480,740 @@ a {
...
+
+  .desk-nav {
+    width: 1350px;
+    height: 100px;
+    padding: 30px 485px;
+    background: #fff;
+    box-shadow: 0 0.5px 0 #dddfe6;
+    justify-content: center;
+    align-items: center;
+  }
```
[the remark this answers is not in the document]
Comment (ksigei) [reply styles/main.css:502]: I have changed all widths to percentage but some doesnt work, do you have an idea?

File: index.html [hunk 4]
```diff
@@ -9,261 +9,392 @@
...
+                 <div class="project-lg">
+                   <div class="project-lg-1">
+                     <h3>Project name goes here</h3> 
+                     
+                     <div class="list-languages">
+                         <ul>
+                             <li><span>HTML/CSS</span></li>
+                             <li><span>Ruby on Rails</span></li>
+                             <li><span>JavaScript</span></li>
+                         </ul>
```
Comment (ShoiraTa) [inline index.html:119]: - [ ] In mobile width the buttons still don't have paddings, Please add paddings in mobile view as well 😉
Screenshot 2022-02-25 at 13 07 19

File: index.html [hunk 5]
```diff
@@ -9,261 +9,392 @@
...
+                        <h3>Get started</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                    </div>
+                    <form action="https://formspree.io/f/xbjwgkqr" method="post">
+                        <input type="text" name="name" class="name" placeholder="Your name" maxlength="30" required>
+                        <input type="email" name="email" class="email" placeholder="Your email" required>
+                        <textarea name="message" class="msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <button class="btn-collabo-desk btn-collabo" type="submit" name="submit">Start collaboration</button>
+        
+                    </form>
```
Comment (ShoiraTa) [inline index.html:375]: - [ ] As requested by the previous reviewer, pleas amke sure to add some padding for every input to be like the design in Figma: 
Screenshot 2022-02-25 at 13 08 57

File: index.html [hunk 6]
```diff
@@ -9,261 +9,392 @@
...
+                         <li><a href="#"><img src="images/github.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/twitter.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/linkedin.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/m-vector.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/fa-vector.png" class="fa-img" alt="image"></a></li>
+                      </ul>        
+                 </div>
+             </div>
+         </footer>      
+    </div>
```
Comment (ShoiraTa) [inline index.html:398]: - [ ] Optional: I recommend to delete extra white space below the footer in mobile view: 
Screenshot 2022-02-25 at 13 10 02

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ksigei,

I totally agree that the design of this template is the most complicated. You really have done a tremendous job!

There are only 2 issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (ksigei) [reply index.html:375]: Hi, I tried this but it does not work, do you have an idea?

Comment (ksigei) [reply index.html:375]: input[type="text"], [type="email"]::placeholder {  
                  
  padding-left: 10px;
  padding-top: 0px; 
} 
textarea::placeholder {  
                  
  padding: 15px; 
} 



I used this code, hope it is okay

Comment (ksigei) [reply index.html:375]: screen-reviewer

Comment (ShoiraTa) [reply index.html:375]: Hey, I can see that you have already solved the issue 👍🏻

Comment (ShoiraTa) [reply index.html:375]: It looks great now ☀️

Comment (ksigei) [reply index.html:375]: Okay

Comment (ksigei) [reply index.html:375]: I have submitted, thanks

**Review by thecodechaser — APPROVED** [review]
## Hi @ksigei 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a title and summary. ✔️


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**